src/qibolab/instruments/qm/components/__init__.py

```diff
@@ -1,2 +1,7 @@
-from .channel import *
-from .configs import *
+from . import configs
+
+# TODO: Fix pycln configurations in pre-commit to preserve the following with no comment
+from .configs import *  # noqa
+
+__all__ = []
+__all__ += configs.__all__
```

Comment:
Why `from .configs import *` is not sufficient here?

Reply:
I need to access `configs.__all__`, because I'm updating the list of symbols exported by the component subpackage (otherwise it would be undeclared).

But `__all__` has a different meaning here, because it will be the `components.__all__`. So, I want the:
- unscoped import (`from .configs import *`) to export the content of the `configs` module
- module import (`from . import configs`) to access the list of exported symbols